ansible/roles/docker-cache/tasks/archlinux.yml

```diff
@@ -0,0 +1,9 @@
+---
+- name: install openebs required packages
+  pacman:
+    name: docker-compose
+    update_cache: true
```

Comment:
The name needs to be refactored to match the task

Reply:
haha whoops! just a blind copy/paste job.

Reply:
this has been updated in [b92e90a](https://github.com/raspbernetes/k8s-cluster-installation/pull/76/commits/b92e90afaf334380363d4b7bafafe70325a8072a)